src/entry.ts

```diff
@@ -151,16 +166,18 @@ class Entry {
    *     [LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry).
    * @returns {Entry}
    */
-  static fromApiResponse_(entry) {
+  // tslint:disable-next-line no-any
+  static fromApiResponse_(entry: any) {
```

Comment:
why are we ignoring the "no-any" rule here. not sure if this is accepted

Reply:
Resolved!